src/main/java/games/strategy/engine/random/RandomStatsDetails.java

```diff
@@ -38,53 +38,53 @@
   }
 
   private static DiceStatistic getDiceStatistic(final IntegerMap<Integer> stats, final int diceSides) {
-    final double m_average;
-    final int m_total;
-    final double m_median;
-    final double m_stdDeviation;
-    final double m_variance;
+    final double average;
+    final int total;
+    final double median;
+    final double stdDeviation;
+    final double variance;
     if (stats.totalValues() != 0) {
       int sumTotal = 0;
-      int total = 0;
+      int localTotal = 0;
```

Comment:
I couldn't think of a better name for this variable, as it is a temporary accumulator that really represents the total.  Extracting a new method to perform the calculation would have solved the problem, but I didn't want to do that in a straightforward variable rename PR.